lib/l1_sender/src/config.rs

```diff
@@ -1,55 +1,20 @@
 use alloy::consensus::constants::GWEI_TO_WEI;
 use serde::{Deserialize, Serialize};
-use smart_config::value::SecretString;
-use smart_config::{DescribeConfig, DeserializeConfig};
 use std::time::Duration;
 
 /// Configuration of L1 sender.
-/// todo: consider renaming to L1Config and using in L1Watcher as well.
-#[derive(Clone, Debug, DescribeConfig, DeserializeConfig)]
-#[config(derive(Default))]
+#[derive(Clone, Debug)]
 pub struct L1SenderConfig {
-    /// Private key to commit batches to L1
-    /// Must be consistent with the operator key set on the contract (permissioned!)
-    // TODO: Pre-configured value, to be removed
-    #[config(alias = "operator_private_key", default_t = "0x026f7f2a55e46437c68e5a75b4a61d11e5fb6987620df61d2cd3400226945720".into())]
-    pub operator_commit_pk: SecretString,
-
-    /// Private key to use to submit proofs to L1
-    /// Can be arbitrary funded address - proof submission is permissionless.
-    // TODO: Pre-configured value, to be removed
-    #[config(default_t = "0x1590cd15674138453a4ae04393c41ea4f21306ef9cf5f2ba326d757d4d97bb8a".into())]
-    pub operator_prove_pk: SecretString,
-
-    /// Private key to use to execute batches on L1
-    /// Can be arbitrary funded address - execute submission is permissionless.
-    // TODO: Pre-configured value, to be removed
-    #[config(default_t = "0x1bf13db0eca84058660ab17913b8f6342361b20847d613b4dc241145620cc593".into())]
-    pub operator_execute_pk: SecretString,
```

Comment:
Let's leave one pk here maybe? We pass it separately from config but makes sense as a part of config IMHO